include/ocpp/common/types.hpp

```diff
@@ -610,7 +610,9 @@ std::ostream& operator<<(std::ostream& os, const CertificateHashDataChain& k);
 
 enum class OcppProtocolVersion {
     v16,
-    v201
+    v201,
+    v21,
+    Unknown,
```

Comment:
I am not quite sure I like adding 'Unknown' to this list. If we don't then I would have to pass optionals through all callbacks to signify if we don't get a valid result from the CSMS.

What would you guys prefer?

Reply:
I think it is fine to add it here